config.py

```diff
@@ -11,21 +14,32 @@ class ConfigException(Exception):
     required_any = config.get('Labels', 'required-labels-any')
     required_all = config.get('Labels', 'required-labels-all')
     banned = config.get('Labels', 'banned-labels')
-    REQUIRED_LABELS_ANY = required_any.split(',') if required_any else None
-    REQUIRED_LABELS_ALL = required_all.split(',') if required_all else None
-    BANNED_LABELS = banned.split(',') if banned else None
     GITHUB_USER = config.get('GitHub', 'user')
     GITHUB_PW = config.get('GitHub', 'password')
 except NoSectionError:
-    raise ConfigException(
-        "Please ensure your config file has a [Labels] and [Github] section.\n"
-        "Did you forget to create a configuration file?\n"
-        "You can do this by running\033[1m cp {0}.template {0} \033[0m".format(config_filename)
-    )
+    required_any = os.environ.get('REQUIRED_LABELS_ANY', None)
+    required_all = os.environ.get('REQUIRED_LABELS_ALL', None)
+    banned = os.environ.get('BANNED_LABELS', None)
+    GITHUB_USER = os.environ.get('GITHUB_USER', None)
+    GITHUB_PW = os.environ.get('GITHUB_PW', None)
+
+REQUIRED_LABELS_ANY = required_any.split(',') if required_any else None
+REQUIRED_LABELS_ALL = required_all.split(',') if required_all else None
+BANNED_LABELS = banned.split(',') if banned else None
 
 
 def get_credentials():
     if GITHUB_USER == '' or GITHUB_PW == '':
         return None
     else:
         return GITHUB_USER, GITHUB_PW
+
+
+if not any([REQUIRED_LABELS_ANY, REQUIRED_LABELS_ALL, BANNED_LABELS]):
```

Comment:
Should this also check for the github credentials?

Reply:
Aren't those optional?

Reply:
No, without credentials the status push will fail (401)

Reply:
ah. will update.